govtool/backend/sql/get-network-metrics.sql

```diff
@@ -1,4 +1,45 @@
-WITH current_epoch AS (
+WITH DRepActivity AS (
+    SELECT
+        drep_activity AS drep_activity,
+        epoch_no AS epoch_no
+    FROM
+        epoch_param
+    WHERE
+        epoch_no IS NOT NULL
+    ORDER BY
+        epoch_no DESC
+    LIMIT 1
+),
+active_drep_boundary_epoch AS (
+    SELECT 
+        epoch_no - drep_activity AS epoch_no
+    FROM
+        DRepActivity
+),
+RankedDRep AS (
+    SELECT
+        dh.raw AS drep_hash_raw,
+        b.epoch_no,
+        dr.deposit,
+        dr.voting_anchor_id,
+        ROW_NUMBER() OVER (PARTITION BY dh.raw ORDER BY b.epoch_no DESC) AS rank
```

Comment:
I'm not sure if dr.tx_id wouldn't be better here instead of b.epoch_no
```suggestion
        ROW_NUMBER() OVER (PARTITION BY dh.raw ORDER BY dr.tx_id DESC) AS rank
```